code/game/objects/effects/spiders.dm

```diff
@@ -190,6 +190,15 @@
 								to_chat(S, "<span class='biggerdanger'>You are a spider who is loyal to [S.master_commander], obey [S.master_commander]'s every order and assist [S.master_commander.p_them()] in completing [S.master_commander.p_their()] goals at any cost.</span>")
 			qdel(src)
 
+/obj/structure/spider/spiderling/decompile_act(obj/item/matter_decompiler/C, mob/user)
+	if(!istype(user, /mob/living/silicon/robot/drone))
```

Comment:
I'm pretty sure nothing uses this other than drones. If you intend to remove it, then remove it--don't add this check here.